com.imageline.flstudio.appdata.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.imageline.flstudio.desktop</id>
```

Comment:
have a look at the id guidelines on https://github.com/flathub/flathub/wiki/AppData-Guidelines the .desktop bit isn't recommended (you'll need to add another bit to say that, but it'll be better overall)